### PR TITLE
fix(mc-scripts): use terser as minifier

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

Uglify-js is not recommended to be used anymore. Also CRA uses terser. Furthermore, uglify-js has issues with hooks. Just google for "Cannot set property 'lastEffect' of null".

### CODE REVIEW

**Comment by tdeekens** [comment]
Will quickly give this a spin cherry-picking it into a pre-canary branch to publish and branch deploy with it.

File: packages/mc-scripts/config/create-webpack-config-for-production.js [hunk 1]
```diff
@@ -120,7 +78,10 @@ module.exports = ({
     // https://medium.com/webpack/webpack-4-mode-and-optimization-5423a6bc597a
     optimization: {
       minimizer: [
-        new UglifyJsPlugin(uglifyConfig),
+        new TerserPlugin({
```
Comment (amine-benselim) [inline packages/mc-scripts/config/create-webpack-config-for-production.js]: woah! this is too simple 👍  😍

Comment (tdeekens) [reply packages/mc-scripts/config/create-webpack-config-for-production.js]: It has some options too but these are the most useful I think. The compression options we can opt-into once we need them I guess. The branch builds fine. Will dig a bit into size comparison before merging.

Comment (tdeekens) [reply packages/mc-scripts/config/create-webpack-config-for-production.js]: Sizes are very similar. Nothing to hold back the migration I think.

**Pull request merged**